ssp-balancer/scripts/proof.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+if [ ! -d bal-mining-scripts ]; then
+    git clone -b feat/custom_proof_script https://github.com/Tesseract-Finance/bal-mining-scripts.git
+    cd bal-mining-scripts
+    git remote add upstream https://github.com/balancer-labs/bal-mining-scripts.git
+    git fetch upstream
+    git merge upstream/master feat/custom_proof_script
+    echo "NETWORK=${1}" > .env
+    npm install
```

Comment:
Do we need to run this setup here? I would imagine we run this script from a docker container and we do the setup when building the docker image.

Reply:
We need to fetch some JSON artifacts from the balancer repo each week and use these to generate the proof.